lost/src/main/java/com/mapzen/android/lost/api/LocationAvailability.java

```diff
@@ -0,0 +1,94 @@
+package com.mapzen.android.lost.api;
+
+import android.content.Intent;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Status on the availability of location data.
+ *
+ * Delivered from LocationCallback registered via
+ * {@link FusedLocationProviderApi#requestLocationUpdates(
+ * LocationRequest,LocationCallback,android.os.Looper) or from a PendingIntent registered via
+ * {@link FusedLocationProviderApi##requestLocationUpdates(LocationRequest,
+ * android.app.PendingIntent). It is also available on demand via
+ * {@link FusedLocationProviderApi#getLocationAvailability()}.
+ */
+public class LocationAvailability implements Parcelable {
+
+  public static final String EXTRA_LOCATION_AVAILABILITY =
+      "com.mapzen.android.lost.EXTRA_LOCATION_AVAILABILITY";
+
+  boolean locationAvailable = false;
+
+  public LocationAvailability(boolean available) {
+    locationAvailable = available;
+  }
+
+  protected LocationAvailability(Parcel in) {
+    locationAvailable = in.readByte() != 0;
+  }
+
+  public static final Creator<LocationAvailability> CREATOR = new Creator<LocationAvailability>() {
+    @Override public LocationAvailability createFromParcel(Parcel in) {
+      return new LocationAvailability(in);
+    }
+
+    @Override public LocationAvailability[] newArray(int size) {
+      return new LocationAvailability[size];
+    }
+  };
+
+  /**
+   * Extracts the LocationAvailability from an Intent.
+   * @param intent
```

Comment:
We should start filling in Javadoc descriptions for params and return values where appropriate. Using `@link` and `@code` tags would also be a nice touch. For example:

```java
  /**
   * Extracts the {@link LocationAvailability} from an {@link Intent}.
   * @param intent an {@link Intent} that may or may not include {@link LocationAvailability}.
   * @return a {@link LocationAvailability} object, or {@code null} if the given {@link Intent} does not contain this data.
   */
```